hdinsight-linux-ssh-publickey-metastore-vnet/azuredeploy.json

```diff
@@ -0,0 +1,230 @@
+            }
+        },
+        "sshUserName": {
+            "type": "string",
+            "metadata": {
+                "description": "These credentials can be used to remotely access the cluster."
+            }
+        },
+        "sshPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The password must be at least 10 characters in length and must contain at least one digit, one non-alphanumeric character, and one upper or lower case letter."
+            }
+        },
+        "existingClusterStorageResourceGroup": {
+            "type": "string",
+            "metadata": {
+                "description": "The resource group name of the storage account to use as the cluster's default storage."
+            }
+        },
+        "existingClusterStorageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the storage account to use as the cluster's default storage."
+            }
+        },
+        "newOrExistingClusterStorageContainerName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the storage container to use."
+            }
+        },
+        "existingHiveMetastoreServerName": {
+            "type": "string",
+            "metadata": {
+                "description": "The fully-qualified domain name of the SQL server to use for the external Hive metastore."
+            }
+        },
+        "existingHiveMetastoreDatabaseName": {
+            "type": "string",
+            "metadata": {
+                "description": "The external Hive metastore's existing SQL database."
+            }
+        },
+        "existingHiveMetastoreUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "The external Hive metastore's existing SQL server admin username."
+            }
+        },
+        "existingHiveMetastorePassword": {
+            "type": "string",
+            "metadata": {
+                "description": "The external Hive metastore's existing SQL server admin password."
+            }
+        },
+        "existingVirtualNetworkResourceGroup": {
+            "type": "string",
+            "metadata": {
+                "description": "The existing virtual network resource group name."
+            }
+        },
+        "existingVirtualNetworkName": {
+            "type": "string",
+            "metadata": {
+                "description": "The existing virtual network name."
+            }
+        },
+        "existingVirtualNetworkSubnetName": {
+            "type": "string",
+            "metadata": {
+                "description": "The existing virtual network subnet name."
+            }
+        }
+    },
+    "variables": {
+        "defaultApiVersion": "2015-06-15-preview",
```

Comment:
This is a GA API version for IaaS resources so it doesn't have preview in it. "2015-06-15"